pkg/kubelet/kubelet_pods.go

```diff
@@ -291,69 +279,83 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 			vol.SELinuxLabeled = true
 			relabelVolume = true
 		}
-		hostPath, err := volumeutil.GetPath(vol.Mounter)
```

Comment:
I am concerned of moving and changing so much code that is not under the feature gate. Was it added alongside the feature?

Ideally we want to minimize the number of potential regressions and keep existing codepaths as close to what they were without the feature gate as possible

Reply:
will update

Reply:
> I am concerned of moving and changing so much code that is not under the feature gate. Was it added alongside the feature?

It just adds a new conditon check for the previous code. It doesn't change the code itself. I add the feature-gate check in line 289. If the feature-gate is disabled, the image will be nil and old logic will be executed.  @SergeyKanzhelev